src/core/ext/transport/cronet/transport/cronet_transport.c

```diff
@@ -51,612 +51,918 @@
 
 #define GRPC_HEADER_SIZE_IN_BYTES 5
 
-// Global flag that gets set with GRPC_TRACE env variable
+#define CRONET_LOG(...)                          \
+  {                                              \
+    if (grpc_cronet_trace) gpr_log(__VA_ARGS__); \
+  }
+
+/* TODO (makdharma): Hook up into the wider tracing mechanism */
 int grpc_cronet_trace = 1;
 
-// Cronet transport object
+enum OP_RESULT {
+  ACTION_TAKEN_WITH_CALLBACK,
+  ACTION_TAKEN_NO_CALLBACK,
+  NO_ACTION_POSSIBLE
+};
+
+/* Used for printing debug */
+const char *op_result_string[] = {"ACTION_TAKEN_WITH_CALLBACK",
+                                  "ACTION_TAKEN_NO_CALLBACK",
+                                  "NO_ACTION_POSSIBLE"};
```

Comment:
I'd recommend having this inside a function with a `switch` instead, so that the chances of the enums and the associated strings going out of sync is minimized.